coalib/parsing/InvalidFilterException.py

```diff
@@ -1,2 +1,8 @@
 class InvalidFilterException(Exception):
-    pass
+    def __init__(self, filter_name=None, available_filters=None):
+        self.filter_name = filter_name
+        self.available_filters = sorted(available_filters)
```

Comment:
I don't think we need to store parameters (filter_name, available_filters) in member variables.

Reply:
@aashutoshrathi @shreyans800755 At least the name maybe can't hurt...